packages/core/admin/admin/src/hooks/useEnterprise/tests/useEnterprise.test.js

```diff
@@ -0,0 +1,80 @@
+import { act, renderHook } from '@testing-library/react';
+
+import { useEnterprise } from '../useEnterprise';
+
+const CE_DATA_FIXTURE = ['CE'];
+const EE_DATA_FIXTURE = ['EE'];
+
+function setup(...args) {
+  return renderHook(() => useEnterprise(...args));
+}
+
+describe('useEnterprise (CE)', () => {
+  beforeAll(() => {
+    window.strapi.isEE = false;
+  });
```

Comment:
this is the default – do you still need this?

Reply:
Sorry it was a leftover is now gone.